### ISSUE TITLE
"No API key found in request"

### ISSUE THREAD

**Issue by Puuhapete1** [cree-py/pynite#7]
Hi, I'm trying to use your code, everything seems to be well, but for some reason I only get 
"No API key found in request" what ever I try.

The same happens even when trying with Postman (program for testing api) and this:
http://api.fortnitetracker.com/v1/my-epic-nicname-here/pc/?TRN-Api-key=my-api-key-here

So it seems that maybe the API is not working at all currently? Or am I doing something wrong?

**Comment by L3NNY0969** [comment]
You need a api key silly

**Comment by batshalregmi** [comment]
well you see its a dead project

**Comment by L3NNY0969** [comment]
Silly

**Comment by erickang21** [comment]
well it's ded af

**Comment by batshalregmi** [comment]
@Puuhapete1 its dead
sorry

**Comment by erickang21** [comment]
maybe try requesting raw api and getting 403 with that brain of yours?

**Comment by batshalregmi** [comment]
no use mine its better

**Comment by L3NNY0969** [comment]
Join my server for some more help @Puuhapete1

**Comment by batshalregmi** [comment]
no i know more python then @L3NNY0969 join mine https://discord.gg/9rc8pVE

**Comment by erickang21** [comment]
no, both their servers are fucking dead. join my server, active, and awesome: https://discord.gg/3Nxb7yZ

**Comment by batshalregmi** [comment]
all he does is ping @everyone so fake

**Comment by batshalregmi** [comment]
i dont ping @everyone

**Comment by L3NNY0969** [comment]
Bish dont listen to this n00bs join  https://bit.ly/2tPxUxm

**Comment by erickang21** [comment]
fuck off both of u I got a custom link: https://discord.me/dbbbothouse

**Comment by batshalregmi** [comment]
fake

**Comment by batshalregmi** [comment]
my bot got more servers

**Comment by erickang21** [comment]
my server got more bots

**Comment by erickang21** [comment]
and people

**Comment by batshalregmi** [comment]
fake

**Comment by erickang21** [comment]
no u

**Comment by IceeMC** [comment]
LOL

**Comment by ravener** [comment]
Lmao

**Comment by batshalregmi** [comment]
they are all fake except me

**Comment by ravener** [comment]
Lol

**Comment by SharpBit** [comment]
lmao wtf is this

**Comment by IceeMC** [comment]
mood

**Comment by ravener** [comment]
Lmao

**Comment by batshalregmi** [comment]
Good old days
